app/views/content/_table_data_description.html.erb

```diff
@@ -1,9 +1,12 @@
 <h1 class="table-header">Showing
     <span class="table-header__param"><%= pagination.first_record %></span> to
     <span class="table-header__param"><%= pagination.last_record %></span> of
-    <span class="table-header__param"><%= pagination.total_results %></span> results
+    <span class="table-header__param"><%= pagination.formatted_total_results %></span> results
+    <% if filter.search_terms? %>
+        for "<span class="table-header__param"><%= filter.search_terms %></span>"
+    <% end %>
     <% if filter.document_type? %>
-        for <span class="table-header__param"><%= filter.document_type %></span>
+        in <span class="table-header__param"><%= filter.document_type %></span>
     <% end %>
-    from <span class="table-header__param"><%= filter.organisation_name %></span>.
+    from <span class="table-header__param"><%= filter.organisation_name %></span>
```

Comment:
Was it intentional to remove the full stop?

Reply:
Yup

Reply:
Maybe I should have read the Trello card first